### PR TITLE
Show info, warn, error by default

### PR DESCRIPTION
**Pull request by tomohiroarakawa**
Show info, warn, and error logs by default

### CODE REVIEW

File: oada-core/docker-compose.yml [hunk 1]
```diff
@@ -130,7 +130,7 @@ services:
       - ./oada-srvc-docker-config.js:/oada-srvc-docker-config.js
     environment:
       - NODE_ENV=${NODE_ENV:-development}
-      - DEBUG=${DEBUG:-""}
+      - DEBUG=${DEBUG:-*:error,*:warn,*:info}
```
Comment (awlayton) [inline oada-core/docker-compose.yml]: I'm not sure we want to have all this on by default?

**Comment by awlayton** [comment]
This looks like a great start, but there are still a lot of `console.log` calls left.

File: oada-core/docker-compose.yml (hunk 1 above)
Comment (tomohiroarakawa) [reply oada-core/docker-compose.yml]: I think microservices should always print those messages by default or you will not notice them unless you turn them on.

Comment (tomohiroarakawa) [reply oada-core/docker-compose.yml]: Each microservice should print one-line info per event

**Comment by tomohiroarakawa** [comment]
There are so many console.log calls. I could not fix them all

File: oada-core/docker-compose.yml (hunk 1 above)
Comment (awlayton) [reply oada-core/docker-compose.yml]: I would say error should definitely be on by default. But In production I'm not sure you need the info messages. maybe warnings.

Comment (tomohiroarakawa) [reply oada-core/docker-compose.yml]: I have changed most of `info` calls to `trace` but maybe it's still excessive. But I would like to see at least one `info` message per request like what most HTTP servers print to an access log

Comment (awlayton) [reply oada-core/docker-compose.yml]: Sounds good

**Comment by abalmos** [comment]
Can we do a massive sed to replace all `console.log` -> `trace`, `console.error` -> `error`, etc?

**Comment by tomohiroarakawa** [comment]
We could do that, but I would probably replace them manually because `console.log` sometimes makes more sense than `info` or `trace`, e.g., when printing usage info.

**Comment by awlayton** [comment]
I agree sometimes you do want `console` calls but I don't think we want any in our oada services. none of them print use info or anything. I like @abalmos idea, except everyone only used `console.log` even for errors and warnings which we would not want to just stick in `trace`.

**Comment by tomohiroarakawa** [comment]
I have replaced most of `console.log` calls in da12f6c

**Comment by abalmos** [comment]
This looks good and would be useful to have. There is more that needs to be done, but merging this effort.

**Comment by tomohiroarakawa** [comment]
Thanks

**Pull request merged**